packages/utilities/src/object.ts

```diff
@@ -1,3 +1,5 @@
+import { Stylesheet } from '@uifabric/merge-styles';
```

Comment:
Again, this import seems strange - it might "accidentally" work here because of what you already have built in your environment... but it shouldn't work in a clean build unless utilities takes on styling as a dep.

Reply:
we already discussed :)